pemFioi/quickpi/blocklyQuickPi_lib.js

```diff
@@ -1670,10 +1684,30 @@ var getContext = function (display, infos, curLevel) {
                 python: ['sleep']
             },
             {
-                id: 'quickpi_cloud',
+                id: 'quickpi_magneto',
                 order: 210,
+                python: ['readMagneticForce', 'computeCompassHeading']
+            },
+            {
+                id: 'quickpi_ir_receiver',
+                order: 211,
+                python: ['readInfraredState', 'readIRMessage']
+            },
+            {
+                id: "quickpi_ir_emitter",
+                order: 212,
+                python: ["setInfraredState", "sendIRMessage"]
+            },
+            {
+                id: "quickpi_potentiometer",
+                order: 213,
+                python: ["readRotaryAngle"]
+            },
+            {
+                id: 'quickpi_cloud',
+                order: 220,
                 python: ['writeToCloudStore','connectToCloudStore','readFromCloudStore']
-            }
+            },
```

Comment:
A comma after the last element of a list is a syntax mistake and will cause some browsers to not accept the script.

Reply:
Fixed in last push.